libs/block-template-html/package.json

```diff
@@ -22,16 +22,15 @@
     "dev": "echo \"Serving at http://localhost:63212\" && http-server -p 63212 ./dev",
     "fix:eslint": "eslint --fix .",
     "format": "prettier --write --ignore-unknown .",
+    "postinstall": "rimraf ./dev/src && lnk -f ./src ./dev",
     "lint:eslint": "eslint --report-unused-disable-directives .",
-    "prepare": "rimraf ./dev/src && lnk -f ./src ./dev",
```

Comment:
`prepare` is [deprecated](https://yarnpkg.com/advanced/rulebook/#packages-should-use-the-prepack-script-to-generate-dist-files-before-publishing) in Yarn Berry